### PR TITLE
Pods 319

### PR DESCRIPTION
**Pull request by suryabalasingh**

### CODE REVIEW

File: app/navigators/RegisterNavigator.scala [hunk 1]
```diff
@@ -74,4 +86,31 @@ class RegisterNavigator @Inject() extends Navigator {
         controllers.routes.SessionExpiredController.onPageLoad()
     }
   }
-}
```
Comment (RyanThompson123) [inline app/navigators/RegisterNavigator.scala]: new line maybe?

File: test/controllers/register/trustees/company/CheckYourAnswersControllerSpec.scala [hunk 2]
```diff
@@ -39,7 +39,7 @@ class CheckYourAnswersControllerSpec extends ControllerSpecBase {
   lazy val companyDetailsSection = AnswerSection(
     Some("messages__checkYourAnswers__section__company_details"),
     Seq(
-      AnswerRow("messages__common__cya__name", Seq(companyName), false, companyDetailsRoute),
+      AnswerRow("messages__common__cya__name", Seq("test company name"), false, companyDetailsRoute),
```
Comment (RyanThompson123) [inline test/controllers/register/trustees/company/CheckYourAnswersControllerSpec.scala]: should be 'test scheme name' not 'test company name'

File: test/navigators/RegisterNavigatorSpec.scala [hunk 3]
```diff
@@ -155,5 +162,107 @@ class RegisterNavigatorSpec extends WordSpec with MustMatchers with PropertyChec
...
+    }
+
+    ".nextPage(SchemeReviewId)" must {
+      "return a 'call' to 'SchemeEstablishedCountryController' page" in {
+        val result = navigator.nextPage(SchemeReviewId, NormalMode)(emptyAnswers)
+        result mustEqual controllers.register.routes.DeclarationDormantController.onPageLoad()
+      }
+    }
   }
-}
```
Comment (RyanThompson123) [inline test/navigators/RegisterNavigatorSpec.scala]: new line

File: test/controllers/register/trustees/company/CheckYourAnswersControllerSpec.scala (hunk 2 above)
Comment (suryabalasingh) [reply test/controllers/register/trustees/company/CheckYourAnswersControllerSpec.scala]: not required :)

**Pull request merged**